aitviewer/renderables/smpl.py

```diff
@@ -107,6 +107,8 @@ def __init__(
         self.poses_left_hand = to_torch(poses_left_hand, dtype=dtype, device=device)
         self.poses_right_hand = to_torch(poses_right_hand, dtype=dtype, device=device)
 
+        self.original_poses = to_torch(original_poses, dtype=dtype, device=device)
```

Comment:
```suggestion
        self.original_poses = to_torch(self.original_poses, dtype=dtype, device=device)
```
In line 144. First assign the value, then convert it to torch.